src/assisted_installer_controller/assisted_installer_controller.go

```diff
@@ -1094,15 +1097,29 @@ func (c controller) sendCompleteInstallation(ctx context.Context, isSuccess bool
 
 // logClusterOperatorsStatus logging cluster operators status
 func (c controller) logClusterOperatorsStatus() {
+	_, _ = c.operatorReport(true)
```

Comment:
can we log it here? and remove this boolean?

Reply:
Done